postpic/particles/_routines.py

```diff
@@ -72,8 +69,14 @@ def histogramdd(data, **kwargs_in):
     edges : list
         A list of D arrays describing the edges for each dimension
     '''
-    kwargs = histogramdd_defs.copy()
-    kwargs.update(kwargs_in)
+    kwshape = kwargs.pop('shape', None)
+    kwshape = 2 if kwshape is None else kwshape  # default value
```

Comment:
Is it actually important to catch the case where a `None` was explicitly passed as `kwargs['shape']`? This might be worth a comment. Otherwise I would suggest ```kwshape = kwargs.pop('shape', 2)``` :)

Reply:
This is important, since `histogramdd(..., shape=None)` and `histogramdd(...)` (shape argument not given) should yield the same result.

Reply:
Maybe add a comment about this, so that no one else gets the idea of replacing this with ```kwshape = kwargs.pop('shape', 2)``` and introduces a bug.

Reply:
done